examples/bond/scripts/deploy.js

```diff
@@ -0,0 +1,115 @@
+const {
+  executeTransaction, convert
+} = require('@algo-builder/algob');
+const { types } = require('@algo-builder/web');
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const storeManagerAccount = deployer.accountsByName.get('alice');
+  const creatorAccount = deployer.accountsByName.get('john');
```

Comment:
these accounts are used in multiple places. Let's create a file `scripts/common/accounts` with a function which will create and return that accounts, eg:
```
export accounts(deployer) {
    return {
        masterAcc: deployer.accountsByName.get('master-account'), ...
    }
}

Reply:
ok